config/locales/en.yml

```diff
@@ -1370,6 +1373,7 @@ en:
   text_work_packages_destroy_confirmation: "Are you sure you want to delete the selected work package(s)?"
   text_work_packages_ref_in_commit_messages: "Referencing and fixing work packages in commit messages"
   text_journal_added: "%{label} %{value} added"
+  text_journal_aggregation_time_explanation: "Combine journals for display if their age difference is less than the specified timespan."
```

Comment:
Probably also point out that this will delay mail delivery, which is the more controversial part of this setting.